### ISSUE TITLE
Archive this repository

### ISSUE THREAD

**Issue by gadomski** [stac-utils/stac-sentinel#5]
It seems like this has been Overtaken By Events, specifically the sentinel stactools packages:

- https://github.com/stactools-packages/sentinel1
- https://github.com/stactools-packages/sentinel1-grd
- https://github.com/stactools-packages/sentinel2
- https://github.com/stactools-packages/sentinel3
- https://github.com/stactools-packages/sentinel5p

If we archive this repo, we might help reduce confusion for newcomers to the ecosystem.

**Comment by matthewhanson** [comment]
Agree, this library was used for sentinel-2 in Earthsearch, but now that's been updated to use stactools_sentinel2.
